curriculum/challenges/english/20-upcoming-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/65687dd4b641fdc67d7de487.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 65687dd4b641fdc67d7de487
+title: Step 29
+challengeType: 20
+dashedName: step-29
+---
+
+# --description--
+
+Move the `print` call below the `number` reassignment.
```

Comment:
I kind of did this in previous step (wrote the reassignment above the `print`). If there's some clever way to direct camper to do it below print, without simply forcing the order in previous step, it could be a good idea.

Reply:
Without forcing it with a test, the only thing I can think of is moving the editable region in the previous lesson to not include the `print`, but that would limit the scope in the editor to an empty line, which I do not think is a better approach.